tofu/geom/_core.py

```diff
@@ -6025,7 +6025,7 @@ def calc_signal(
         reflections=True,
         coefs=None,
         ind=None,
-        out=object,
+        returnas=object,
```

Comment:
much more explicit variable name, especially since we tend to use 'out' to store temporary output inside functions.
It as the case in the other method by the way (calc_signal_from_Plasma2D()), out was redefined at line 6254